File: wear/src/main/java/io/github/francoiscampbell/manytimes/TimerDisplayActivity.java
package io.github.francoiscampbell.manytimes;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class TimerDisplayActivity extends Activity {

    private TextView mTextView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_display);
        mTextView = (TextView) findViewById(R.id.text);
    }
}

File: wear/src/main/kotlin/io/github/francoiscampbell/manytimes/TimerDisplayActivity.kt
package io.github.francoiscampbell.manytimes

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class TimerDisplayActivity : Activity() {

    private var mTextView: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_display)
        mTextView = findViewById(R.id.text) as TextView
    }
}
